src/inputs/DateFields/DateFieldBase.tsx

```diff
@@ -86,9 +88,11 @@ export function DateFieldBase(props: DateRangeFieldBaseProps | DateSingleFieldBa
     onEnter,
     defaultOpen,
     mode,
+    isRangeFilterField = false,
     ...others
   } = props;
 
+  const isRangeMode = mode === "range";
```

Comment:
Keeping things DRY.

Reply:
Nice! When I was doing this, I had so many issues trying to store local variables and keep TS happy. I ended up just grabbing everything off `props`, but glad we can at least store this one.